index.html

```diff
@@ -20,54 +20,68 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
+    <div class="menu-bar hidden-menu">
+      <button class="close-btn">
+              <i class="fa fa-times"></i>
+      </button>
+      <div class="menu-mobile">
```

Comment:
- [ ] Correct font-size 👌 But let's fix the font-family and font-weight like on the Figma template as suggested by the previous reviewer.

![image](https://user-images.githubusercontent.com/63473584/215590197-a2d3d75b-4029-4766-96c7-8fafa955692a.png)